src/main/resources/templates/partials/navbar.ftl

```diff
@@ -7,16 +7,18 @@
 
   <!-- Links -->
   <div class="navbar-nav">
-    <a class="nav-item nav-link" href="/home">Home</a>
-    <a class="nav-item nav-link" href="/questions">Questions</a>
-    <a class="nav-item nav-link" href="/profile">Profile</a>
+    <!-- <a class="nav-item nav-link" href="/home">Home</a> -->
+    <a class="nav-item nav-link" href="/question">Questions</a>
+    <a class="nav-item nav-link" href="/ask-question">Ask a Question</a>
+    <!-- <a class="nav-item nav-link" href="/profile">Profile</a> -->
   </div>
 
   <!-- Search Bar -->
   <form class="form-inline ml-auto">
     <input class="form-control mr-sm-2" type="search" placeholder="Search..." aria-label="Search">
     <button class="btn btn-success my-2 my-sm-0" type="submit"><img src="img/search.png"></button>
   </form>
-  <a href="#" class="btn btn-info my-2 my-sm-0 ml-2 justify-content-end"><img src="img/settings.png"></a>
-  <a href="/" class="btn btn-danger my-2 my-sm-0 ml-2 justify-content-end"><img src="img/logout.png"></a>
-</nav>
\ No newline at end of file
+  <a href="#" class="btn btn-info my-2 my-sm-0 ml-2 justify-content-end"><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABkAAAAZCAYAAADE6YVjAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAAAKySURBVEhLzZVdaI5hGMfftWkbLaJQZDnhxPLdZpyQHVIOZk05YEq02ebMRygHOKD5SFGGsNR8tHIg7UApaTTsxAnT8lEO1RQHXn7/y38vj/fZG2vFv37d9/W/rvu5n3f3/VzL/FfKZrO7v1nMv0K1U+MnHtoFmzw/A82RGIt40cl+4HpbIeLnsNjzHXAuEog1RcTtcJx5se10eYOHcBlewlm8mYx66DCUq46xFoZgA8yCu/AAeqGz4EYUXIGrfrMKuAif4BQsc9nIm6+DW/AZDurBMJH5U2hxab5I1sEQxVNsySv1NFXkJ3mq+QL4AJW20kVBB1xz+MdiTSno3DbbGl0UzYBhfk2ZrRDedGjHP8rYBHE+IyJeAa8cFhaFLdDhMESsP+Nb6IS9cBtewHyXhIj7IW5gnkjo+l2HAdAtyhUy1y97B8tthYi3QJ8O3Ja8nfARHsEFcvVOxW3RV9zIsIgxcdDEbVrgMCH8Z7DSYYhn6DOohv0wYPvHJp7micJjsMdhQvg3ocFhQvhV8HMTAvUitYrtUMOeRU4pp0PucZgTNSX4g7DQlmrL8dcy6s/fDf1ORVI/rxnOgw64zqlYCDrkrbZGNjgN3bZCxIdB1/gkbIPExciJRAN0OQwRz4PHoAfcgNcep7pEG08gHmScY2t0UVgJXyBxAVistrEK6slV2c4JL/qZNrOVLgrUk+7BPlt/Jdb1wBGH6aJAX/R9iLtPrFaxJJIpok5nU6uXU8x8GryB1VGQJpInoJdF6qa6gjoDtXS1/18PvgwO4enfgc7nDuijnW1v9P7FwmIK1N7VrtVNo5hRt0+doMLxGtCHOJc1Jd7wPehKt6mmoLzRLki0a+I+qPG8FX7vb3qRRodjE5tf4iEHGJcy6vo2OTV+4qEb2eCJYK5flf6h/RtlMt8BfFRJSv0q+7IAAAAASUVORK5CYII="></a>
```

Comment:
This change reverts the usage of files within the img tag and goes back to using base64 encoded images.

Reply:
Resolved